src/Symfony/Component/Console/Application.php

```diff
@@ -564,6 +564,10 @@ public function addCommand(callable|Command $command): ?Command
                 ->setDescription($attribute->description ?? '')
                 ->setHelp($attribute->help ?? '')
                 ->setCode($command);
+
+            foreach ($attribute?->usages ?? [] as $usage) {
```

Comment:
The attribute is never null in this case:

```suggestion
            foreach ($attribute->usages as $usage) {
```